src/ansys/fluent/core/utils/deprecate_args.py

```diff
@@ -0,0 +1,31 @@
+"""Deprecates old argument and replaces them with corresponding new argument."""
+
+import functools
+import warnings
+
+
+def deprecate_arguments(
```

Comment:
This only works for a single argument, right?
```suggestion
def deprecate_argument(
```

Reply:
Done. Thank you @raph-luc.